exonum/src/events/tests.rs

```diff
@@ -259,6 +268,33 @@ fn test_network_big_message() {
     assert_eq!(e2.wait_for_disconnect(), first);
 }
 
+#[test]
+fn test_network_max_message_len() {
+    let first = "127.0.0.1:17200".parse().unwrap();
+    let second = "127.0.0.1:17201".parse().unwrap();
+
+    let m1 = raw_message(15, ConsensusConfig::DEFAULT_MESSAGE_MAX_LEN as usize - 1000);
+    let m2 = raw_message(16, ConsensusConfig::DEFAULT_MESSAGE_MAX_LEN as usize + 1000);
+
+    let e1 = TestEvents::with_addr(first);
+    let e2 = TestEvents::with_addr(second);
+
+    let mut e1 = e1.spawn();
+    let mut e2 = e2.spawn();
+
+    e1.connect_with(second);
+    e2.wait_for_connect();
+
+    e2.connect_with(first);
+    e1.wait_for_connect();
+
+    e1.send_to(second, m1.clone());
+    assert_eq!(e2.wait_for_message(), m1);
+
+    e2.send_to(first, m2.clone());
+    assert!(e1.receives_error());
```

Comment:
nit: can be replaced with `e1.wait_for_event().is_err()`

Reply:
Oh, jeez, indeed...